### ISSUE TITLE
Nozzle types

### ISSUE THREAD

**Issue by DerkHBos** [3DCP-TUe/SaladSlicer#14]
Circular, rectangular, automatic set height

**Comment by arjendeetman** [comment]
What is the idea? Is this same as later geometry visualisation?

**Comment by DerkHBos** [comment]
Yes it can be used in geometry visualization. Also possibly to set the layer height or calculate the extruder ratio
